src/main/scala/org/squbs/sample/RequestHandler.scala

```diff
@@ -0,0 +1,56 @@
+package org.squbs.sample
```

Comment:
Actually, I do not like the name of the file `RequestHandler`.  I wonder if I should rename it as `SampleWellKnownActor.scala` or `SampleActor.scala`.